### ISSUE TITLE
query by id

### ISSUE THREAD

**Issue by hoclun-rigsep** [taspinar/twitterscraper#190]
Is it possible to query by the number that comes at the end of the url when you 'copy link to tweet'?

**Comment by taspinar** [comment]
This would mean you are querying a single tweet?

**Comment by hoclun-rigsep** [comment]
Yes; I'd like to write a script I can activate right after copying the link to the tweet.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by taspinar** [comment]
@hoclun-rigsep 
I dont think that is possible. [Here ](https://twitter.com/search-advanced?lang=en&lang=en)you can see the fields you can use for searching for specific tweets. 
So you could narrow it down by specifying the username, a specific daterange and the words /hashtags contained in the tweet, but you can not search for a specific tweet by its id.

**Comment by hoclun-rigsep** [comment]
Do you think the `since_id` and `max_id` params described below might do it?

https://developer.twitter.com/en/docs/tweets/search/api-reference/get-search-tweets

**Comment by rezemika** [comment]
It seems that the HTML tags for a unique tweet are in fact the sames as the tags for search results. So, given a user and a tweet ID, it's possible to get the tweet URL and to do a request to it. It would return the tweet and its replies. It would require to write a dedicated function, but it's feasible. ;)
